packages/desktop-gui/src/settings/configuration.jsx

```diff
@@ -50,7 +50,29 @@ const normalizeWithoutMeta = (value = {}) => {
 const ObjectLabel = ({ name, data, expanded, from, isNonenumerable }) => {
   const formattedData = formatValue(data)
 
-  return (
+  const localData = useLocalStore(() => ([formattedData]))
```

Comment:
It keeps changing the values in the UI, but the actual app's state is still the same. I tried importing these methods from `mobx-react`, but I haven't figured it out yet. Any tips will be appreciated. [Source](https://mobx-react.js.org/state-local#what-about-global-store)